From 776876a9d9e6f094c3b3680e84734579e7a7f529 Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mervin.fansler@bric.ku.dk>
Date: Thu, 5 Sep 2024 09:59:06 +0200
Subject: [PATCH] manual ucrt libs

---
 src/Makevars.ucrt | 14 ++++----------
 1 file changed, 4 insertions(+), 10 deletions(-)

diff --git a/src/Makevars.ucrt b/src/Makevars.ucrt
index 8ff3402..fbbb02e 100644
--- a/src/Makevars.ucrt
+++ b/src/Makevars.ucrt
@@ -10,16 +10,10 @@ endif
 
 PKG_CPPFLAGS = -I.
 
-ifeq (,$(shell pkg-config --version 2>/dev/null))
-  PKG_CPPFLAGS += -I$(LOCAL_SOFT)/include/cairo -I$(LOCAL_SOFT)/include/freetype2
-  LIBSHARPYUV = $(or $(and $(wildcard $(R_TOOLS_SOFT)/lib/libsharpyuv.a),-lsharpyuv),)
-  PKG_LIBS += -lcairo -lfontconfig -lintl -liconv -lexpat -lfreetype -lharfbuzz \
-              -lpixman-1 -ltiff -lwebp $(LIBSHARPYUV) -llzma -lzstd -ljpeg -lpng -lbz2 \
-              -lz -lgdi32 -lmsimg32
-else
-  PKG_CPPFLAGS += $(shell pkg-config --cflags cairo libtiff-4 libjpeg)
-  PKG_LIBS += $(shell pkg-config --libs cairo libtiff-4 libjpeg)
-endif
+PKG_LIBS += -L$(CYGWIN_PREFIX)/Library/lib -lcairo -lfontconfig -lintl -liconv -lexpat -lfreetype -lharfbuzz \
+            -lpixman-1 -ltiff -lwebp -llzma -lzstd -ljpeg -lpng -lbz2 \
+            -lz -lgdi32 -lmsimg32
+PKG_CPPFLAGS += -I$(CYGWIN_PREFIX)/Library/include -I$(CYGWIN_PREFIX)/Library/include/cairo -I$(CYGWIN_PREFIX)/Library/include/freetype2
 
 PKG_LIBS += $(GRAPHAPP_LIB)
 PKG_CPPFLAGS += $(XTRA_PKG_CPPFLAGS)
